build-system/tasks/karma.conf.js

```diff
@@ -156,8 +156,10 @@ module.exports = {
   client: {
     mocha: {
       reporter: 'html',
-      // Longer timeout on Travis; fail quickly at local.
-      timeout: process.env.TRAVIS ? 10000 : 2000,
+      // Allow tests to run for up to 5 seconds locally and on Travis.
+      // TODO(rsimha-amp): Reduce local run timeout to 2s after large tests are
```

Comment:
@kmh287 is working on #9477 to move the bind tests to integration/, at which point we'll no longer need a 5s timeout for local tests. I'll change that back to 2s ASAP.

@cramforce, are you okay with the Travis time limit for unit tests being brought down to 5s, or do you see any reason to leave that at 10s?